### ISSUE TITLE
Release a v2 with new exports

### ISSUE THREAD

**Issue by icholy** [lukeed/hexoid#7]
The v1.0.0 release causes issues with webpack because the commonjs & ESM modules don't export the same API.

* https://github.com/ladjs/superagent/issues/1786
* https://github.com/auth0/node-auth0/issues/798
* https://stackoverflow.com/questions/78246983/nextjs-push-files-to-mongodb-hexoid-is-not-a-function-error
* https://github.com/node-formidable/formidable/issues/960
* https://github.com/node-formidable/formidable/issues/871

I noticed that this has been fixed in master, but hasn't been published. Can this happen?

**edit:** Here's a minimal repro of the problem:

**main.js**
``` js
const hexoid = require('hexoid');

hexoid(25);
```
**webpack.config.js**
``` js
module.exports = {
	target: 'node',
	entry: './main.js',
	optimization: {
		minimize: false
	}
}
```

Steps:
``` sh
webpack
node dist/main.js
```
Output:
```
/home/icholy/src/playground/hexoid-webpack/dist/main.js:90
hexoid(25);
                ^

TypeError: hexoid is not a function
    at /home/icholy/src/playground/hexoid-webpack/dist/main.js:90:17
    at Object.<anonymous> (/home/icholy/src/playground/hexoid-webpack/dist/main.js:92:12)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49
```

**Comment by bompus** [comment]
Master does seem to work properly. For now, I am using ES and `import hexoid from 'hexoid';`, but I believe on master, it would be a breaking semver major change to `import { hexoid } from 'hexoid';`

**Comment by icholy** [comment]
@bompus yep, that's why it needs to be a v2 release.

**Comment by mobeigi** [comment]
@lukeed Any chance we can get this resolved? Would be much appreciated.

**Comment by lukeed** [comment]
sorry, not sure how/why the branch was left unsync'd 

fixed now -- hexoid@2.0.0 is now available

**Comment by icholy** [comment]
Thanks!
